include/glow/CodeGen/MemoryAllocator.h

```diff
@@ -105,6 +158,9 @@ class MemoryAllocator {
   /// allocation at \p address.
   bool hasHandle(uint64_t ptr) const;
 
+  /// \returns the segment currently associated with the \p handle.
+  Segment getSegment(Handle handle) const;
```

Comment:
Nit: just checking if it should be Segment or rather `const Segment &`?

Reply:
Changed to `const Segment &`.